### DISCUSSION TITLE
Data Stating Duration for Larger Batch Sizes

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/deepmd-kit#1684] [Q&A]
With larger data sets, in training, data stating can take a very long time especially for larger batch sizes.  Is there any way to alleviate this?  Thank you very much for your time.

**Comment by wanghan-iapcm** [comment]
One may consider reducing the `data_stat_nbatch` 
see https://docs.deepmodeling.com/projects/deepmd/en/latest/train/train-input.html

**Comment by npiroozan** [comment]
Thank you very much indeed.  I would like to understand what is happening in this phase of the simulation.  What happens in data_stating?  Thank you very much.

**Reply by njzjz** [reply]
It does two things:
(1) Normalizes the environment matrices;
(2) Determines the initial atomic energy biases (trainable) for each atom type.
